lib/tasks/metro.rake

```diff
@@ -47,4 +47,13 @@ namespace :metro do
       )
     end
   end
+
```

Comment:
Yeah, I think that's fine. Migrations for this kind of thing depend on Models and DB state and other things which have a tendency to be fragile over time, so a one-time task seems OK.